python/OpenMCMaterial.py

```diff
@@ -55,16 +55,15 @@ def __write_mass_fraction(material, nuclide, mass_frac):
 # material xml element
 def write_openmc_material(MaterialCard, material_tree):
     matid = str(MaterialCard.material_number)
-    name = "Material " + str(matid)
+    matname = str(MaterialCard.material_name)
```

Comment:
a little more nuance here I think, the original intent here is that it gets a name corresponding to an mcnp mateiral id. In this instance files translated will now not have any material id. So I think something more like
```
if MaterialCard.material_name not "":
   name = str(materialCard.material_name)
else
   name = str(MaterialCard.material_number)
```
This way if name is set it gets used, otherwise its generated from the material number

Reply:
Never mind, following our conversation.